build/yamls/antrea-windows-with-ovs.yml

```diff
@@ -1,5 +1,24 @@
 apiVersion: v1
 data:
+  Install-OVSDriver-Containerd.ps1: |
```

Comment:
I feel not necessary to highlight "Containerd" in the name by now.

Reply:
Sure, I will update all related script name.